TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -38,21 +38,25 @@
     private LinkedList<BaseServerObject> serverQueue;
     private LinkedList<BaseProxyObject> proxyQueue;
 
-    private Map<Integer, Integer> recentlyUsedPorts;
-
     private final ScheduledExecutorService scheduler;
 
     private Map<String, FileTailer> logTailers;
 
     private boolean startingServer = false;
+    private final Integer startPortServer = 41000;
```

Comment:
Change to `private final static int SERVER_PORT_START`, `SERVER_PORT_MAX` and accordingly for proxies. 
Move all the lines with static variables to one section (e.g. at the top) of the variables.

Reply:
Nobody needs more than 250 proxies, so we can change the values accordingly.